### ISSUE TITLE
Support simple protocol imports and units_of()

### ISSUE THREAD

**Issue by jonc125** [ModellingWebLab/weblab-fc#106]
Test by creating a Python version of https://chaste.cs.ox.ac.uk/trac/browser/chaste/projects/FunctionalCuration/test/TestS1S2Protocol.hpp

- [x] Test `units_of` first with `test_clamping.py:test_clamping_with_variable_units` - can be separate PR
- [x] Then LR91 model tests basic import functionality
    - [x] Top-level import of a protocol, merging model interface
    - [x] Overriding stimulus current definition - should be done by #97 but check!
    - [x] The `units_of` construct in a larger equation: `0 :: units_of(oxmeta:membrane_stimulus_current_amplitude)`
- [x] Noble model tests with time units conversion - shouldn't really need anything further implemented after the above, just check it works!
- [x] I'm not sure how much of the manifest is currently implemented in the Python FC code. Check, and open a separate issue for that if necessary - don't consider it part of this one. --> See #148 

Blocked by #95. Part of #23.

**Comment by MichaelClerx** [comment]
Did the `units_of` part, but what next @jonc125 ? Imports are tested in `test_nested_protocols`, as far as I can see

**Comment by jonc125** [comment]
There's the first bit of the description:
> Test by creating a Python version of https://chaste.cs.ox.ac.uk/trac/browser/chaste/projects/FunctionalCuration/test/TestS1S2Protocol.hpp

Only nested protocol imports are already tested, not the import usage in `test_S1S2.txt`.

**Comment by MichaelClerx** [comment]
Noble s1s2 test uses unit conversion _within equations_, see discussion on #144

**Comment by MichaelClerx** [comment]
When should `units_of` be resolved, by the way? Is _before_ any model modifications?

**Comment by jonc125** [comment]
Typically it'll be used for variables not given units in the protocol (since otherwise you'd know the units). But if applied to a variable that had specified units I think using those the protocol sets would be most expected?

--
Written on a smartphone; please excuse brevity!
________________________________

**Comment by MichaelClerx** [comment]
We'd better check that then
